config/default/manager_visibility_patch.yaml

```diff
@@ -13,3 +13,10 @@ spec:
           - containerPort: 8082
             name: visibility
             protocol: TCP
+          volumeMounts:
+          - mountPath: /visibility
+            name: visibility
+      volumes:
```

Comment:
Drop a small comment explaining why this is needed.

Reply:
Added.